Dockerfile

```diff
@@ -114,24 +115,24 @@ RUN apk add --no-cache --virtual=.build-deps build-base && \
      apk del .build-deps && \
      mkdir -p /tmp/.cache
 
+# TODO: we should remove this (we were supposed to get rid of it in June 2022)
 COPY entrypoint.sh /entrypoint.sh
 RUN chmod +x /entrypoint.sh
 
 # Let the user know how their container was built
 COPY Dockerfile /Dockerfile
 
 # Get semgrep-core from step1
-COPY --from=semgrep-core /semgrep/semgrep-core/_build/default/src/cli/Main.exe /usr/local/bin/semgrep-core
+COPY --from=semgrep-core-container /src/semgrep/semgrep-core/_build/default/src/cli/Main.exe /usr/local/bin/semgrep-core
 
+# ???
 ENV SEMGREP_IN_DOCKER=1 \
      SEMGREP_VERSION_CACHE_PATH=/tmp/.cache/semgrep_version \
      SEMGREP_USER_AGENT_APPEND="Docker"
 
-WORKDIR /src
```

Comment:
hmm I've removed this because I dunno what it's for (it was not commented), but I think we might rely on this
for commands we tell people to run.
Still does not look like I broke CI so there was no test for it.